docs/en/serverless/logging/monitor-datasets.mdx

```diff
@@ -0,0 +1,51 @@
+---
+id: serverlessObservabilityMonitorDatasets
+slug: /serverless/observability/monitor-datasets
+title: Monitor log data set quality
```

Comment:
```suggestion
title: Monitor log dataset quality
```
This should probably be one word?
There are more occurrences in the rest of the doc.

Reply:
Thanks for pointing that out. It's actually the opposite where I need to change the other occurrences to "data set". Lisa pointed out that the [branding guide](https://brand.elastic.co/302f66895/p/194a3b-writing-style-guide/t/889c93) has it as two words.